python/observability-online-eval/langgraph/computer-use-agent/main.py

```
@@ -0,0 +1,180 @@
+
+workflow.add_edge(START, "process_input")
+workflow.add_conditional_edges("process_input", route_after_processing_input)
+workflow.add_edge("respond", END)
+workflow.add_edge("computer_use_agent", END)
+
+graph = workflow.compile()
+graph.name = "Price Finder"
+
+
+@langgraph_agent(name="cua-v1")
+async def ask_agent(messages):
```

Ruff (0.11.9)

Missing return type annotation for public function `ask_agent`

(ANN201)